spec/factories/address.rb

```diff
@@ -0,0 +1,24 @@
+# t.string :number
+# t.string :street
+# t.string :settlement
+# t.string :postal_code
+# t.string :municipality
+# t.string :state
+#
+# t.timestamps
+# <Address id: 3, number: nil, street: nil,
+# settlement: "Machado Sur",
+# postal_code: "22703", municipality: "Playas de Rosarito",
+# state: "Baja_California", created_at: "2021-05-26 15:31:41.616405000 +0000",
+# updated_at: "2021-05-26 15:31:41.616405000 +0000", denounce_id: 14>
```

Comment:
If all these comments are not necessary you can remove them, so that the code looks cleaner.